plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/AnalyticsOptInFieldEditor.java

```diff
@@ -0,0 +1,73 @@
+package com.google.cloud.tools.eclipse.preferences;
+
+import org.eclipse.jface.preference.BooleanFieldEditor;
+import org.eclipse.jface.preference.FieldEditor;
+import org.eclipse.jface.preference.IPreferenceStore;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Group;
+import org.eclipse.swt.widgets.Link;
+
+public class AnalyticsOptInFieldEditor extends FieldEditor {
+
+  private Group group;
+  private BooleanFieldEditor optInStatusEditor;
+
+  /**
+   * @param name the name of the preference this field editor works on
+   * @param parent the parent of the field editor's control
+   */
+  public AnalyticsOptInFieldEditor(String name, Composite parent) {
+    group = new Group(parent, SWT.SHADOW_OUT);
+    group.setText(Messages.FIELD_EDITOR_ANALYTICS_GROUP_TITLE);
```

Comment:
You should configure the `Group`'s layout.  It's likely easiest to do:
`GridLayoutFactory.swtDefault().applyTo(group)`.  That should correct the inter-widget spacing of the widgets within the group.